Markdown/syntax_test_markdown.md

```diff
@@ -1927,7 +1927,7 @@ var_dump(expression);
 |^^^ meta.paragraph meta.code-fence.definition.end.regexp - markup
 |^^ punctuation.definition.raw.code-fence.end
 
-```bash
+``` bash
```

Comment:
it would probably be a good idea to add an assertion that the whitespace isn't scoped as `punctuation` or `constant.other.language-name`, and that "`bash`" is still scoped correctly as `constant.other.language-name`

Reply:
Good suggestion. I moved this to a new test. Not sure if the assertion for the whitespace-part is okay.